__tests__/action-types.js

```diff
@@ -0,0 +1,27 @@
+import { testActionTypes } from '../src/action-types-array';
+import { TypeError, UnicityError } from '../src/errors';
+// ------------------------------------ //
+const arrayToTest = ['one', 'two', 'three'];
+const arrayToTest2 = [...arrayToTest, 77];
+const arrayToTest3 = [...arrayToTest, 'two'];
+const PREFIX = 'SomePrefix';
+// ------------------------------------ //
+describe('hey', () => {
```

Comment:
if no good description is apparent, better to leave off describe block.

Reply:
I use them as separators since I find it prettier than blank lines, but for sure, I'll remove them all.

Reply:
Done.